apps/molgenis-components/src/components/task/Task.vue

```diff
@@ -57,14 +57,14 @@ export default defineComponent({
           .then((data) => {
             this.task = data._tasks[0];
             this.$emit("taskUpdated", this.task);
+            this.loading = false;
           })
           .catch((error) => {
             console.log(JSON.stringify(error));
```

Comment:
The whole component is unclear to me; 
what is does 'loading' indicate  ? ( the task is running or 'we are fetching the task status ? )
what about the catch ( or finally ) should the 'loading' flag be set to false ?
maybe replace the while loop with a for/break loop of avoid endless looping in case the backend fails

Reply:
You are correct. Added the other this.loading = false;